Cargo.toml

```diff
@@ -16,6 +16,9 @@ serde = { version = "1.0", features = ["derive"] }
 napi-derive = { version = "2", default-features = false, features = ["strict", "compat-mode"] }
 quiche = { version = "0.17.1", features = ["boringssl-boring-crate", "boringssl-vendored"] }
 boring = "2.1.0"
+# FIXME: temp for testing
+log = { version = "0.4", features = ["std"] }
+ring = "0.16"
```

Comment:
What's this for?

Reply:
It's used by the example code in `./examples/test_example.rs`. I think we can remove log but ring is needed to generate the token. But maybe we can stub that out as well.

Reply:
I think the examples should be self contained into a subdirectory and shouldn't be part of the main scaffolding.

Reply:
Removed.